nanoFramework.System.Net.Http/Http/Headers/MediaTypeHeaderValue.cs

```diff
@@ -0,0 +1,131 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+namespace System.Net.Http.Http.Headers
+{
+    /// <summary>
+    /// Represents a media type used in a Content-Type header as defined in the RFC 2616.
+    /// </summary>
+    /// <remarks>
+    /// The MediaTypeHeaderValue class provides support for the media type used in a Content-Type header as defined in RFC 2616 by the IETF.
+    /// An example of a media-type would be "text/plain; charset=iso-8859-5".
+    /// </remarks>
+    public class MediaTypeHeaderValue
+    {
+        private const string _CharSetLabel = "charset=";
+        private const int _CharSetLabelLenght = 8;
+
+        /// <summary>
+        /// Gets or sets the character set.
+        /// </summary>
+        /// <value>The character set.</value>
+        public string CharSet { get; set; }
+
+        /// <summary>
+        /// Gets or sets the media-type header value.
+        /// </summary>
+        /// <value>The media-type header value.</value>
+        /// <remarks>
+        /// The media-type is used in the Content-Type and Accept header fields in order to provide open and extensible data typing and type negotiation.
+        /// </remarks>
+		public string MediaType { get; set; }
```

Comment:
issue with indentation, ctrl + k + d should help :-)

Reply:
still shows issue with the methods indentations, maybe just github rendering:
![Uploading image.png…]()

Reply:
did it 10x.... just remove and copy back the line